Emby.Dlna/Configuration/DlnaOptions.cs

```diff
@@ -17,7 +17,7 @@ public DlnaOptions()
             BlastAliveMessages = true;
             SendOnlyMatchedHost = true;
             ClientDiscoveryIntervalSeconds = 60;
-            AliveMessageIntervalSeconds = 1800;
+            AliveMessageIntervalSeconds = 180;
```

Comment:
You sure we want this? I believe it was set that high because of increased load

Reply:
It's disabled by default and shouldn't be necessary in most cases. If I remember correctly the spec says they can/should be sent every multiple hundred milliseconds, so I guess the new value should be better than the old.